src/content/learn/typescript.md

```diff
@@ -456,7 +456,7 @@ We recommend the following resources:
 
  - [The TypeScript handbook](https://www.typescriptlang.org/docs/handbook/) is the official documentation for TypeScript, and covers most key language features.
 
- - [The TypeScript release notes](https://devblogs.microsoft.com/typescript/) covers a each new features in-depth.
+ - [The TypeScript release notes](https://devblogs.microsoft.com/typescript/) covers each new features in-depth.
```

Comment:
```suggestion
 - [The TypeScript release notes](https://devblogs.microsoft.com/typescript/) covers new features in-depth.
```

I noticed this typo and was about to make a PR but saw that you'd noticed it first—thanks! This sentence should either read "covers new features" or "covers each new feature" to have the correct pluralization.

Reply:
Good catch, added it.